connector_easypost_tracker/models/stock_picking_tracking_event.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+from openerp import fields, models
+from openerp.addons.connector.unit.mapper import (
+    mapping,
+    only_create,
+)
+from openerp.addons.connector_easypost.backend import easypost
+from openerp.addons.connector_easypost.unit.backend_adapter import (
+    EasypostCRUDAdapter,
+)
+from openerp.addons.connector_easypost.unit.import_synchronizer import (
+    EasypostImporter,
+)
+from openerp.addons.connector_easypost.unit.mapper import (
+    EasypostImportMapper,
+    eval_false,
+)
+from .stock_picking_tracking_location import (
+    StockPickingTrackingLocationImporter,
+)
+
+_logger = logging.getLogger(__name__)
+
+
+class EasypostStockPickingTrackingEvent(models.Model):
+    """ Binding Model for the Easypost StockPickingTrackingEvent"""
+    _name = 'easypost.stock.picking.tracking.event'
+    _inherit = 'easypost.binding'
+    _inherits = {'stock.picking.tracking.event': 'odoo_id'}
+    _description = 'Easypost StockPickingTrackingEvent'
+    _easypost_model = 'Tracker'
+
+    odoo_id = fields.Many2one(
+        comodel_name='stock.picking.tracking.event',
+        string='Stock Picking Tracking Event',
+        required=True,
+        ondelete='cascade',
+    )
+
+    _sql_constraints = [
+        ('odoo_uniq', 'unique(backend_id, odoo_id)',
```

Comment:
Nope, `backend_id` and `easypost_id` are the only unique constraint imposed there.